scripts/output/projects/FSEC_dietaryIndicators.R

```diff
@@ -0,0 +1,51 @@
+# |  (C) 2008-2021 Potsdam Institute for Climate Impact Research (PIK)
+# |  authors, and contributors see CITATION.cff file. This file is part
+# |  of MAgPIE and licensed under AGPL-3.0-or-later. Under Section 7 of
+# |  AGPL-3.0, you are granted additional permissions described in the
+# |  MAgPIE License Exception, version 1.0 (see LICENSE file).
+# |  Contact: magpie@pik-potsdam.de
+
+# --------------------------------------------------------------
+# description: Extract population-level dietary information and consumption data from a MAgPIE run
+# comparison script: FALSE
+# ---------------------------------------------------------------
+
+# Version 1.00 - Michael Crawford
+# 1.00: first working version
+
+library(lucode2)
+library(magpie4)
+
+print("Start FSEC dietary indicators output runscript")
+
+############################# BASIC CONFIGURATION #######################################
+
+if(!exists("source_include")) {
+
+    title       <- NULL
+    outputdir   <- NULL
+
+    # Define arguments that can be read from command line
+    readArgs("outputdir", "title")
+}
+#########################################################################################
+
+print(paste0("Script started for output directory: ", outputdir))
```

Comment:
```suggestion
message("Script started for output directory: ", outputdir)
```

`message` also allows you to get rid of the `paste0` call